apache-rat-core/src/main/resources/org/apache/rat/plain-rat.xsl

```diff
@@ -19,59 +19,94 @@
+            <xsl:call-template name="statistic">
+                <xsl:with-param name="name"><xsl:value-of select="@name"/></xsl:with-param>
+                <xsl:with-param name="count"><xsl:value-of select="@count"/></xsl:with-param>
+                <xsl:with-param name="description"><xsl:value-of select="@description"/></xsl:with-param>
+                <xsl:with-param name="leadin"><xsl:choose>
+                    <xsl:when test='@approval="false"'><xsl:text>! </xsl:text></xsl:when>
+                    <xsl:otherwise><xsl:text>  </xsl:text></xsl:otherwise>
+                </xsl:choose></xsl:with-param>
+            </xsl:call-template>
+        </xsl:for-each>
+
+        <xsl:call-template name="subsection">
+            <xsl:with-param name="title">Licenses detected</xsl:with-param>
+        </xsl:call-template>
+
+        <xsl:for-each select='descendant::licenseName'>
+            <xsl:value-of select='concat($newline, @name, ": ", @count, " ")'/>
+        </xsl:for-each>
+
+        <xsl:call-template name="subsection">
+            <xsl:with-param name="title">License Categories detected</xsl:with-param>
+        </xsl:call-template>
+
+        <xsl:for-each select='descendant::licenseCategory'>
+            <xsl:value-of select='concat($newline, @name, ": ", @count, " ")'/>
+        </xsl:for-each>
+
+        <xsl:call-template name="subsection">
+            <xsl:with-param name="title">Document Types detected</xsl:with-param>
+        </xsl:call-template>
+
+        <xsl:for-each select='descendant::documentType'>
+            <xsl:value-of select='concat($newline, @name, ": ", @count, " ")'/>
+        </xsl:for-each>
+
+        <xsl:if test="descendant::resource[license/@approval='false']">
+            <xsl:value-of select="concat($newline, $newline)" />
+            <xsl:call-template name="section">
+                <xsl:with-param name="title">Files with unapproved licenses</xsl:with-param>
+            </xsl:call-template>
+
+            <xsl:for-each select='descendant::resource[license/@approval="false"]'>
+                <xsl:value-of select='concat($newline, "  ", @name)'/>
+            </xsl:for-each>
+
+        </xsl:if>
+
+        <xsl:if test="descendant::resource[@type='ARCHIVE']">
+            <xsl:value-of select="concat($newline, $newline)" />
+            <xsl:call-template name="section">
+                <xsl:with-param name="title">Archives</xsl:with-param>
+            </xsl:call-template>
+
+            <xsl:for-each select='descendant::resource[@type="ARCHIVE"]'>
+                <xsl:value-of select='concat($newline, " ", @name)'/>
+            </xsl:for-each>
+        </xsl:if>
+        <xsl:value-of select="concat($newline, $newline)" />
+        <xsl:call-template name="section">
+            <xsl:with-param name="title">Detail</xsl:with-param>
+        </xsl:call-template>
+        <xsl:text>
+  The line following documents with unapproved licenses will start with a '!'
```

Comment:
This line makes me stumble in the report. Can it be simplified to:
* "Lines starting with '!' indicate that the document contains an unapproved license" OR
* "Documents found to contain unapproved licenses are marked as '!'"

If I have a look at the reports being generated during the build of the feature branch, I interpret the '!' to mean "this document contains an unapproved license".

WDYT?

Reply:
There seems to be a misunderstaning - I only had the feeling that the sentence in the report itself can be simplified to be more concise/clearer. Nothing more - no functional change or change in the report functionality.